civis/futures.py

```diff
@@ -495,35 +484,27 @@ class _ContainerShellExecutor(_CivisExecutor):
     inc_script_names: bool, optional
         If ``True``, a counter will be added to the ``script_name`` to create
         the script names for each submission.
+    **kwargs:
+        Additional keyword arguments will be passed
+        directly to ``scripts.post_containers``.
 
     See Also
     --------
     civis.APIClient.scripts.post_containers
     """
     def __init__(self, docker_image_name="civisanalytics/datascience-python",
                  docker_image_tag="latest",
-                 repo_http_uri=None,
-                 repo_ref=None,
-                 git_credential_id=None,
                  script_name=None,
                  required_resources=None,
```

Comment:
We should remove the script name one. This is clearly a case where name is better since it follows the API. The name of the executor is not really a thing anyways. If people are using this, they must have a fair bit of background already.